codemap/api-hot.md

```
@@ -329,17 +329,17 @@ when WRITING TESTS; they are not feature-code frequency signal.
-- `modify :: (value -> value) -> ConcurrentVar value -> Task w Unit`  <!-- 88 test call sites -->
-- `containing :: value -> Task w (ConcurrentVar value)`  <!-- 84 test call sites -->
-- `peek :: ConcurrentVar value -> Task w value`  <!-- 71 test call sites -->
+- `modify :: (value -> value) -> ConcurrentVar value -> Task w Unit`  <!-- 89 test call sites -->
+- `containing :: value -> Task w (ConcurrentVar value)`  <!-- 85 test call sites -->
+- `peek :: ConcurrentVar value -> Task w value`  <!-- 72 test call sites -->
 - `get :: ConcurrentVar value -> Task w value`  <!-- 39 test call sites -->
 - `set :: value -> ConcurrentVar value -> Task w ()`  <!-- 15 test call sites -->
 - `new :: forall value w. Task w (ConcurrentVar value)`  <!-- 14 test call sites -->
 - `swap :: value -> ConcurrentVar value -> Task w value`  <!-- 3 test call sites -->
 
 ## Uuid
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)